charts/jobs/templates/job.yaml

```diff
@@ -14,9 +14,16 @@ spec:
       annotations:
         checksum/secret: {{ include (print $.Template.BasePath "/secret.yaml") $ | sha256sum | trunc 63 }}
         checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") $ | sha256sum | trunc 63 }}
+        {{- range $key, $value := $spec.annotations }}
+        {{ $key }}: {{ $value | quote }}
+        {{- end }}
     spec:
       serviceAccountName: {{ $spec.serviceAccountName | default "default" }}
       {{- if $spec.init }}
+      {{- with $spec.securityContext }}
```

Comment:
hmm. ok but it does not work here.
While this is a nice way to obtain defaults, it does not work inside this section of yaml. `$spec.init` refers to the **initContainer** section and I don't want to scope the `$spec.securityContext` object into this entire block.

Reply:
well, I think the initContainer may be given a securityContext, but if not then not giving the root securityContext is fine, as it will then be tested for policies which can be excluded in the initContainer's annotations.
So it will become just
```
{{- if $spec.init }}
{{- with $spec.init | get "securityContext" nil }}
...
```
Inside the initContainer spec

Reply:
It does not matter which security context is used here anymore as we cannot annotate individual containers, so both are excluded due to the `otomi:tools` image not running with a known UID `user` directive.

Reply:
I can adjust the template to accept initContainer securityContext as well as pod level securityContext and we will disable annotation if we refactor the `tools` docker image.

Reply:
> It does not matter which security context is used here anymore as we cannot annotate individual containers, so both are excluded due to the `otomi:tools` image not running with a known UID `user` directive.

I think it IS possible to specify per container: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/

So a job should be able to specify per container.

And what does `otomi/tools` image have to do with this? A job can be any image